django/db/migrations/operations/special.py

```diff
@@ -62,10 +62,11 @@ class RunSQL(Operation):
     by this SQL change, in case it's custom column/table creation/deletion.
     """
 
-    def __init__(self, sql, reverse_sql=None, state_operations=None):
+    def __init__(self, sql, reverse_sql=None, state_operations=None, hints=None):
         self.sql = sql
         self.reverse_sql = reverse_sql
         self.state_operations = state_operations or []
+        self.hints = hints or {}
 
     def deconstruct(self):
         kwargs = {
```

Comment:
Maybe add a note that we are deliberately not deconstructing `hints` here because we don't auto-generate the operations? Otherwise somebody is getting confused at some point. (Same for `RunPython`)